interface/system/config_backup.rst

```diff
@@ -24,22 +24,27 @@ e.g. see: :ref:`v3_to_v4`.
 The following state information is saved as part of a backup
 
 * Users and Groups
-* Samba configuration and exports
+* Samba configuration and exports (**Note**: only the global Samba configuration will be restored, share level custom configuration entries will be backed up into the file, but are as of yet not being restored).
 * NFS configuration and exports
 * Schedule task configuration
 * Service configurations (ntp, smartd etc..)
-* Service status (on / off)
+* Service status (on/off)
 * Scheduled task configuration
 * Rock-on configuration
 
-*Note: as of Rockstor-3.9.2-56, AFP configuration and exports are no-longer
-supported.*
+.. note::
+  **Addtional Notes:**
 
-Since Rockstor can dynamically detect  :ref:`Pool <pools>`,
+  * For Samba exports: as of Rockstor 5.0.14-0 only the global Samba configuration will be restored share level custom configuration entries will be saved into the configuration backup file but are not yet being restored.
```

Comment:
I'm thinking of not specifying the Rockstor version here as it might make it appear like we used to support that and decided to deprecate it. I also wonder if mentioning that the export-level custom config is in the config is extra here. How about something akin to the following?
"""
While the configuration of the Samba service itself will be fully restored, the *custom configuration* of Samba exports will not be restored as Rockstor does not yet support restoring this information.
"""

Reply:
I thought of keeping the version in there, as we're usually not incredibly diligent in updating documentation after the fact. 
I like the way you distinguished the between the service level config vs. export 
I wanted to highlight that the info is still stored in the backup file, so all is not lost, if somebody attempts the restore and then realizes that the custom config is not there (you know, the usual way, break the thing first and then read the instructions :) )

But I can see your point only wanting to have version information alongside deprecated features. So how about this merged and augmented version (I didn't want to use the word restore too many times in the same sentence).

```
* For Samba exports: the configuration of the Samba service itself will be fully restored, including the *global custom configuration*. However, the *custom configuration* of Samba exports will only be saved into the configuration file, as Rockstor does not yet support restoring this information.
```

Let me know, if that works.